src/main/kotlin/no/nav/syfo/domain/ManglendeMedvirkningVurdering.kt

```diff
@@ -0,0 +1,73 @@
+package no.nav.syfo.domain
+
+import no.nav.syfo.aktivitetskrav.api.DocumentComponentDTO
+import java.time.LocalDate
+import java.time.OffsetDateTime
+import java.util.UUID
+
+interface IManglendeMedvirkningsVurdering {
+    val uuid: UUID
+    val personident: Personident
+    val veilederident: String
```

Comment:
Denne burde ha en egen type. Regner med at veileder id'er er på et visst format

Reply:
Vi har ikke gjort det før, men enig i at det er en forbedring. Er vel 7 tegn der første er en bokstav og 6 neste er tall.

Reply:
Kan sette opp en egen oppgave for det slik at vi får bekreftet at dette alltid er formatet, og det ikke blokkerer denne 😊